### PR TITLE
Remove shutdown target from stream

### PR DESCRIPTION
**Pull request by jonasherzfeld**

### CODE REVIEW

File: include/nadjieb/mjpeg_streamer.hpp [hunk 1]
```diff
@@ -77,7 +79,7 @@ class MJPEGStreamer : public nadjieb::utils::NonCopyable {
         nadjieb::net::HTTPRequest req(message);
         nadjieb::net::OnMessageCallbackResponse cb_res;
 
-        if (req.getTarget() == shutdown_target_) {
+        if (shutdown_target_ != "" && req.getTarget() == shutdown_target_) {
```
Comment (m-haase) [inline include/nadjieb/mjpeg_streamer.hpp]: why not shutdown_target_.empty() ?

File: single_include/nadjieb/mjpeg_streamer.hpp [hunk 2]
```diff
@@ -885,7 +887,7 @@ class MJPEGStreamer : public nadjieb::utils::NonCopyable {
         nadjieb::net::HTTPRequest req(message);
         nadjieb::net::OnMessageCallbackResponse cb_res;
 
-        if (req.getTarget() == shutdown_target_) {
+        if (shutdown_target_ != "" && req.getTarget() == shutdown_target_) {
```
Comment (m-haase) [inline single_include/nadjieb/mjpeg_streamer.hpp]: why not shutdown_target_.empty() ?

**Review by m-haase — CHANGES_REQUESTED** [review]

File: include/nadjieb/mjpeg_streamer.hpp (hunk 1 above)
Comment (jonasherzfeld) [reply include/nadjieb/mjpeg_streamer.hpp]: done

File: single_include/nadjieb/mjpeg_streamer.hpp (hunk 2 above)
Comment (jonasherzfeld) [reply single_include/nadjieb/mjpeg_streamer.hpp]: done

**Review by m-haase — APPROVED** [review]

**Pull request merged**